stl/inc/xstring

```diff
@@ -2641,6 +2645,37 @@ private:
         }
     }
 
+#if _HAS_CXX20
+public:
+    // Move assign from a buffer, used by basic_stringbuf
+    void _Move_assign_from_buffer(_Elem* _Right, size_type _Size, size_type _Res) noexcept {
+        _Tidy_deallocate();
+        _Right[_Size]  = _Elem();
+        auto& _My_data = _Mypair._Myval2;
+        _Construct_in_place(_My_data._Bx._Ptr, pointer_traits<pointer>::pointer_to(*_Right));
+        _My_data._Mysize = _Size;
+        _My_data._Myres  = _Res;
+    }
+
+    // Release to a buffer, or allocate a new one if in small string mode
+    tuple<_Elem*, size_type, size_type> _Release_to_buffer() {
+        tuple<_Elem*, size_type, size_type> _Result;
+        auto& _My_data = _Mypair._Myval2;
+        if (_My_data._Large_string_engaged()) {
+            _Result           = make_tuple(_Unfancy(_My_data._Bx._Ptr), exchange(_My_data._Mysize, size_type()),
+                exchange(_My_data._Myres, size_type()));

```

Comment:
The non-`_Ugly` function calls `make_tuple` and `exchange` should be `_STD` qualified.